### ISSUE TITLE
Behind Corporate Proxy

### ISSUE THREAD

**Issue by krprasadreddy** [kevinohara80/nforce#93]
I'm not able to authenticate with Salesforce as it refuses to connect.
I'm behind my corporate proxy.How do I set my proxy to bypass it ?

**Comment by krprasadreddy** [comment]
This is the message : Error: connect ECONNREFUSED

**Comment by kevinohara80** [comment]
Ok, I've been meaning to do this for a while and I just released it in 1.2.0...

I've added the ability to pass additional parameters down to the underlying request module. In request, you can specify a proxy. So in the nforce request, just define a `requestOpts` hash with the proxy info...
```js
org.query({ 
  oauth: oauth, 
  query: 'SELECT .... FROM Account',
  requestOpts: {
    proxy: {...}
  }
})
```

I don't have a proxy to test with so give it a shot and let me know.

More info on proxying in request can be found [here](https://github.com/request/request#proxies)
